tests/sympc/protocol/aby3/aby3_test.py

```diff
@@ -0,0 +1,57 @@
+# third party
+import numpy as np
+import pytest
+import torch
+
+from sympc.encoder import FixedPointEncoder
+from sympc.protocol import ABY3
+from sympc.protocol import Falcon
+from sympc.session import Session
+from sympc.session import SessionManager
+from sympc.tensor import MPCTensor
+from sympc.tensor import ReplicatedSharedTensor
+
+
+def test_share_class() -> None:
+    assert ABY3.share_class == ReplicatedSharedTensor
+
+
+def test_session() -> None:
+    protocol = ABY3("semi-honest")
+    session = Session(protocol=protocol)
+    assert type(session.protocol) == ABY3
+
+
+def test_invalid_security_type():
+    with pytest.raises(ValueError):
+        ABY3(security_type="covert")
+
+
+def test_invalid_parties_trunc(get_clients) -> None:
+    parties = get_clients(2)
+    session = Session(parties=parties)
+
+    with pytest.raises(ValueError):
+        ABY3.truncate(None, session)
+
+
+def test_trunc1(get_clients) -> None:
```

Comment:
MAYBE: A test where base is 2?

Reply:
Yeah , tests where base is not 2 right.Good catch,I forgot to add  that scenario.